src/plugins/stringStyle.js

```diff
@@ -0,0 +1,32 @@
+/* @flow */
+import createRule from '../utils/createRule'
+import type {JssStyle, RuleOptions, Rule} from '../types'
+
+const nlOrSemi = /\n|;/
```

Comment:
Also i think this will fail with base64 urls that might have the token `;` in them or any property with a string or function `()` with the token inside of it.

You can do a lookahead to make sure there are no string or function tokens `", ', (, )` but since you can chain css rules in one line that might produce false positives.

i.e

```css
color: red; background: url(http://google.com?s=;); color: rgba(0,0,0);
```